### ISSUE TITLE
Updating quantities

### ISSUE THREAD

**Issue by mazeau** [ReactionMechanismGenerator/RMG-Py#1335]
Quantities stopped working and I got 
```
ValueError: ufunc <ufunc 'positive'> not supported by quantities
                please file a bug report at https://github.com/python-quantities
```

but uninstalled and reinstalled from pip

```
pip install --upgrade git+git://github.com/python-quantities/python-quantities.git@master
```

and now it works.

https://github.com/python-quantities/python-quantities/issues/128

Should we upgrade version in conda channel?

**Comment by mliu49** [comment]
Sure, we can definitely update quantities in the RMG conda channel. @mazeau, are you running into this with current RMG-Py or other code?

**Comment by mazeau** [comment]
on the cat branch I believe

**Comment by mliu49** [comment]
I updated the version in the conda channel, but it seems to have broken one of the quantity tests. Will look into the cause.

**Comment by mliu49** [comment]
Ok, a bug was introduced in quantities v0.12.0 (the version I added to our conda channel), and was fixed in v0.12.1. I'll test the newer version locally this time before updating the conda package.

**Comment by mliu49** [comment]
I updated the quantities package in the RMG channel to v0.12.1 a few weeks ago, so this should be resolved.
